corehq/apps/reports/formdetails/case_metadata.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import absolute_import, unicode_literals
+
+import re
+
+from corehq.apps.app_manager.app_schemas.case_properties import (
+    get_parent_type_map,
+)
+from corehq.apps.app_manager.const import USERCASE_TYPE
+from corehq.apps.app_manager.models import (
+    AdvancedForm,
+    AdvancedModule,
+    Form,
+    FormAction,
+    FormActionCondition,
+    Module,
+)
+from corehq.apps.data_dictionary.util import get_case_property_description_dict
+from corehq.apps.reports.formdetails.readable import (
+    AppCaseMetadata,
+    FormQuestionResponse,
+)
+
+
+class AppCaseMetadataBuilder(object):
+    def __init__(self, domain, app):
+        self.domain = domain
+        self.app = app
+        self.meta = AppCaseMetadata()
+
+    def case_metadata(self):
```

Comment:
**super**nit: This is the only method whose name is a noun. Could it be a property, or `get_case_metadata()` like the function that calls it?